tests/common_tools.py

```diff
@@ -153,12 +153,7 @@ def is_using_jinja2_template(response):
 
 
 def setup_conference_with_typical_fares(start=date(2019, 7, 8), end=date(2019, 7, 14)):
```

Comment:
might be good to change this to 2020 while you're at it. 
Also – might be good to put those dates in settings.py

Reply:
I agree with both points - but would prefer to keep it out of scope of this PR in case this was to cause any other tests to fail.